assets/javascripts/discourse/components/recurring-trigger.js.es6

```diff
@@ -0,0 +1,9 @@
+import Component from "@ember/component";
+
+export default Component.extend({
+  actions: {
+    yolo() {
```

Comment:
🤣

Reply:
oops I moved recurring to another PR, seems like I forgot few parts :p